cpp-package/example/inference/README.md

```diff
@@ -30,34 +30,116 @@ To build examples use following commands:
 
 This directory contains following examples. In order to run the examples, ensure that the path to the MXNet shared library is added to the OS specific environment variable viz. **LD\_LIBRARY\_PATH** for Linux, Mac and Ubuntu OS and **PATH** for Windows OS.
 
-### [inception_inference.cpp](<https://github.com/apache/incubator-mxnet/blob/master/cpp-package/example/inference/inception_inference.cpp>)
-
-This example demonstrates image classification workflow with pre-trained models using MXNet C++ API. The command line parameters the example can accept are as shown below:
-
-```
-./inception_inference --help
+## [imagenet_inference.cpp](<https://github.com/apache/incubator-mxnet/blob/master/cpp-package/example/inference/imagenet_inference.cpp>)
+
+This example demonstrates image classification workflow with pre-trained models using MXNet C++ API. Now this script also supports inference with quantized CNN models generated by Intel® MKL-DNN (see this [quantization flow](https://github.com/apache/incubator-mxnet/blob/master/example/quantization/README.md)). By using C++ API, the latency of most models can get **2%~17%** speedup compared with current Python implementation. The below tables show accuracy and latency of several CNN models.
+
+The following models have been tested on Linux systems. And 50000 images are used to collect the following accuracy numbers.
```

Comment:
I would expect that C++ inference should have the same accuracy as python inference. But seems the data here is different from https://github.com/apache/incubator-mxnet/tree/master/example/quantization. I guess these two table might use different versions of MXNet or GluonCV. But it's still confusing.

Reply:
Actually, I have tested accuracy for C++ and Python locally. There are no difference. I think the numbers in [page]( https://github.com/apache/incubator-mxnet/tree/master/example/quantization) maybe too old and need to be updated.